fireapp/lib/presentation/volunteer_information/volunteer_information.dart

```diff
@@ -0,0 +1,204 @@
+
+class VolunteerInformationPage extends StatefulWidget {
+  final String volunteerId;
+
+  const VolunteerInformationPage({
+    super.key,
+    required this.volunteerId,
+  });
+
+  @override
+  State createState() => _VolunteerInformationState();
+}
+
+class _VolunteerInformationState extends FireAppState<VolunteerInformationPage>{
+
+  @override
+  VolunteerInformationViewModel viewModel = GetIt.instance.get();
+
+  @override
+  void initState() {
+    super.initState();
+    viewModel.getVolunteerInformation(widget.volunteerId);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BasicWrapper(
+      page: RequestStateWidget.stream<VolunteerInformation>(
+        state: viewModel.volunteerInformation,
+        retry: () => viewModel.getVolunteerInformation(widget.volunteerId),
+        child: (_, volunteerInformation){
+          return Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+                SizedBox(height: 12.0),
+                Text(
+                  'Volunteer Information',
+                  style: TextStyle(
+                    fontSize: 24.0,
+                    fontWeight: FontWeight.bold,
+                    color: Colors.blue,
+                  ),
+                ),
+                SizedBox(height: 12.0),
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      'Volunteer ID: ${volunteerInformation.ID}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Name: ${volunteerInformation.firstName} ${volunteerInformation.lastName}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Email: ${volunteerInformation.email}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
+                    Text(
+                      'Volunteer Mobile No: ${volunteerInformation.mobileNo}',
+                      style: TextStyle(
+                        fontSize: 18.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.blue,
+                      ),
+                    ),
+                    SizedBox(height: 12.0),
```

Comment:
If you want to space every widget by 12, don't do this. At the end of the widget array, add `.spacedBy(12)` and import the extension.